client/sites-dashboard/components/sites-site-item-thumbnail.tsx

```diff
@@ -8,6 +8,8 @@ import { useSiteStatus } from '../hooks/use-site-status';
 
 const NoIcon = styled.div( {
 	fontSize: 'xx-large',
+	textTransform: 'uppercase',
+	userSelect: 'none',
```

Comment:
I added this so it feels like an image to the user, getting out of the browser's text selection range.